npm/qsharp/ux/circuit-vis/formatters/gateFormatter.ts

```diff
@@ -359,15 +387,15 @@ const _x = (metadata: Metadata, _: number): SVGElement => {
 const _ket = (label: string, metadata: Metadata): SVGElement => {
   const { x, targetsY, width } = metadata;
   const gate = _unitary(
-    `|${label}〉`,
+    `|${label}⟩`,
```

Comment:
It is hard to tell by eye-balling it. Are these the characters we want to use for the bar and angle-bracket? I see you've changed the bars in the comment above but not here, and you changed the bracket here but not above.

Reply:
See from line 330 in util.ts. I checked all the files and made them consistent.

Reply:
Actually, I see you have a mathChars table in utils.ts now. We could use that instead to enforce consistency.

Reply:
I thought it might just be a little unreadable/busy to use some property of an import interpolated everywhere we have a special char, but could try and see how it looks.

Reply:
I think it does indeed make it hard to read when using those references, so I'm fine with them not being used like this.